python/paddle/v2/fluid/layers/control_flow.py

```diff
@@ -397,9 +397,40 @@ def complete(self):
 
 
 def lod_rank_table(x, level=0):
-    """
-    This function creates an operator for creating a LOD_RANK_TABLE
-    using the input x.
+    """LoD Rank Table Operator. Given an input variable `x` and a LoD level,
+    this layer creates a LodRankTable object. A LoDRankTable object contains a
+    list of bi-element tuples and each tuple consists of an index and a length.
+    For given level's LoD information, the index is the sequence position and
```

Comment:
It's not easy to understand what is `level's LoD information`.

Reply:
Done.